wormhole/file_transport.go

```diff
@@ -184,14 +184,14 @@ type fileTransport struct {
 	appID           string
 }
 
-func (t *fileTransport) connectViaRelay(otherTransit *transitMsg) (net.Conn, error) {
+func (t *fileTransport) connectViaRelay(myTransit *transitMsg, otherTransit *transitMsg) (net.Conn, error) {
 
 	successChan := make(chan net.Conn)
 	failChan := make(chan string)
 
 	var count int
 
-	for _, relay := range otherTransit.HintsV1 {
+	for _, relay := range myTransit.HintsV1 {
```

Comment:
@vu3rdd but we do not combine otherTransit with myTransit, we use only our fallback url here.